src/govuk/settings/_links.scss

```diff
@@ -4,43 +4,62 @@
 
 /// Enable new link styles
 ///
+/// If enabled, the link styles will change. Underlines will:
```

Comment:
This uses a slightly different format to the other mixins ("If you enable this, a thing will happen" here vs "Does something to a thing" elsewhere). Best to make it consistent one way or the other.

Reply:
```suggestion
/// Makes underlines:
```

Reply:
@m-green How's this ^ suggestion - is it a bit bare? Also, should we delete "be" from the first bullet so the sentence as a whole does not read awkwardly ("Makes underlines be consistently thinner...")?

Reply:
'Makes underlines' seems to work! But yep you'd need to delete 'be' from the bullet.

Reply:
Hi Eoin - following our chat, 'enable' makes sense but I'd try to make it active if possible - eg "If you enable this variable...". (Or even better "If you set this variable to `true`..." to make it super-explicit).

Reply:
@36degrees Just talked to @m-green and we're going to revisit this issue another day. We're going to leave the wording as you originally suggested.